src/adapter/aws-lambda/handler.ts

```diff
@@ -83,7 +98,7 @@ const createRequest = (event: APIGatewayProxyEvent | APIGatewayProxyEventV2) =>
 
   const requestInit: RequestInit = {
     headers,
-    method: event.httpMethod,
+    method: event.httpMethod ?? event.requestContext.http?.httpMethod,
```

Comment:
I tried it, but `event.requestContext.http?.httpMethod` is `undefined`. So, `method` must be:

```ts
const method = 'httpMethod' in event ? event.httpMethod : event.requestContext.http.method
```